test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -279,11 +279,11 @@ class test_ucp_sockaddr : public ucp_test {
     {
         ucs_time_t time_limit = ucs_get_time() + ucs_time_from_sec(UCP_TEST_TIMEOUT_IN_SEC);
 
-        while ((receiver().get_num_eps() == 0) && !is_failed() &&
+        while ((receiver().get_num_eps() == 0) && (err_handler_count == 0) &&
```

Comment:
what is reason for this change?

Reply:
When server can't create endpoint to the client, the client gets rejected error code. is_failed returns true for unreachable err code

Reply:
maybe need to change the naming here? (e.g is_failed() to check all errs, is_unreachable() to check unreachable err code)?